account_move_group_restriction/models/account_account.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class AccountAccount(models.Model):
+    _inherit = "account.account"
+
+    security_group_ids = fields.Many2many(
+        comodel_name="res.groups",
+        relation="account_account_res_groups_rel",
+        column1="account_id",
+        column2="group_id",
+        string="Allowed User Groups",
```

Comment:
```suggestion
        string="Access Groups",
        groups="account.group_account_manager",
```
I think ''Access Groups' sounds more precise.